app/controllers/openscap_proxies_controller.rb

```diff
@@ -2,7 +2,7 @@ class OpenscapProxiesController < ApplicationController
   before_action :find_proxy, :only => [:openscap_spool]
 
   def openscap_spool
-    last_error = @smart_proxy ? find_spool_error : nil
+    last_error = @smart_proxy ? @smart_proxy.statuses[:openscap].spool_status : nil
```

Comment:
could we rename `last_errors` to `spool_errors` or just `error` please? that we we don't have to rename in future when we add more error types :-)